### ISSUE TITLE
meta-data-online cellnumber

### ISSUE THREAD

**Issue by INFINITY1993** [biofuture/Ublastx_stageone#77]
Hi, 
I am wondering what dose the "CellNumer" in the "meta_data_online.txt" mean? Does it represent the putative bacterial cell number in the metagenomic sample? 
And why  is the "#of16Sreads" lower than "CellNumer"? as I can imagine there are more than one copy of 16s genes in a single bacterial cell.
Thanks in advance!
![1655282984053](https://user-images.githubusercontent.com/63569349/173785246-c1f80742-419b-4b2c-aaae-4902101a2f24.png)

**Comment by biofuture** [comment]
Yes, you can refer to the paper on how to calculate the numbers.  It represents how many cells are equivalently sequenced in this metagenomics data.

**Comment by INFINITY1993** [comment]
Thanks！
